lib/perl/Genome/InstrumentData/AlignmentResult.pm

```diff
@@ -1094,7 +1098,7 @@ sub _create_bam_index {
     return 1;
 }
 
-sub _create_bam_flagstat {
+sub create_bam_flagstat {
```

Comment:
I agree. I mostly avoided this in the current PR because I was wary of callers somehow depending on the current behavior (updating the flagstat file?). It seems unlikely but I didn't want to add a whole lot of complications to this PR in interests of getting it out soon to solve RT tickets.

I will do a bit of research to see what all utilizes this code.